content/en/synthetics/guide/browser-test-self-maintenance.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Why are browser tests self-maintaining?
+kind: guide
+further_reading:
+    - link: 'synthetics/browser_tests'
+      tag: 'Documentation'
+      text: 'Configure a Browser Test'
+    - link: '/synthetics/browser_tests/actions'
+      tag: 'Documentation'
+      text: 'Create Browser Test Steps'
+    - link: 'https://www.datadoghq.com/blog/test-creation-best-practices/'
+      tag: 'Blog'
+      text: 'Best practices for creating end-to-end tests'
+
+---
+
+Flakiness is one of the biggest pain points associated with end-to-end testing. Typically, oftentimes, end-to-end tests would fail, not because of an actual application issue, but because the frontend team performed some changes causing the identifier used by your test not to work anymore.
+
+## How do browser tests target elements?
+
+Datadog’s Synthetic browser tests address this pain point through an in-house algorithm that leverages a set of locators to target elements. When a test searches for a specific element to interact with (e.g., a checkout button), instead of looking at an element with a specific XPath or a specific CSS selector, browser tests actually use several different points of reference to help find it (XPath, text, classes, what other elements are near it, etc.). These points of reference are turned into a set of locators, each uniquely defining the element.
```

Comment:
```suggestion
Datadog’s Synthetic browser tests address this pain point through an in-house algorithm that leverages a set of locators to target elements. When a test searches for a specific element to interact with (for example, a checkout button) instead of looking at an element with a specific XPath or a specific CSS selector, browser tests use several different points of reference to help find it (XPath, text, classes, what other elements are near it, etc.). These points of reference are turned into a set of locators, each uniquely defining the element.
```